components/searchable_user_list/searchable_user_list.tsx

```diff
@@ -53,59 +54,59 @@ class SearchableUserList extends React.PureComponent {
         focusOnMount: false,
     };
 
-    constructor(props) {
-        super(props);
+    private userList: React.RefObject<UserList>;
+    private nextTimeoutId: number;
 
-        this.nextTimeoutId = 0;
+    constructor(props: Props & IntlProps) {
+        super(props);
 
         this.state = {
             nextDisabled: false,
         };
+
+        this.nextTimeoutId = 0;
+        this.userList = createRef();
     }
 
     componentDidMount() {
         this.focusSearchBar();
     }
 
-    componentDidUpdate(prevProps) {
+    componentDidUpdate(prevProps: Props) {
         if (this.props.page !== prevProps.page || this.props.term !== prevProps.term) {
-            this.refs.userList.scrollToTop();
+            this.userList.current?.scrollToTop();
         }
     }
 
     componentWillUnmount() {
         clearTimeout(this.nextTimeoutId);
     }
 
-    nextPage = (e) => {
-        e.preventDefault();
-
+    nextPage = () => {
         this.setState({nextDisabled: true});
-        this.nextTimeoutId = setTimeout(() => this.setState({nextDisabled: false}), NEXT_BUTTON_TIMEOUT);
+        setTimeout(() => this.setState({nextDisabled: false}), NEXT_BUTTON_TIMEOUT);
```

Comment:
Did you intend to remove the use of `nextTimeoutId` here?

Reply:
nextTimeoutId expects a number so I removed this, now I can see that this is not the solution, an idea here would be great.

Reply:
`setTImeout` returns a `number`, so leaving this alone should be fine.